data/bunrodea/bunrodea ships.txt

```diff
@@ -0,0 +1,496 @@
+# Copyright (c) 2021 by Amazinite
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+ship "Kaiken"
```

Comment:
Iirc, these were supposed to have cloaks, right? I think they did before, but don't at the moment. It sounds like these are the ships in the intro missions too.

Reply:
They had cloaks, but I removed them. Felt like it was a bit much, particularly after bringing the strength of all the Bunrodea ships and weapons down a few pegs.

The Sasumata still has a cloaking device.

Reply:
So would the two ships in the intro be Sasumata's then? Or custom Kaiken's?

Reply:
It's only one ship in the intro now.